### ISSUE TITLE
Training Loop

### ISSUE THREAD

**Issue by Jeli04** [nsnod/CTorch#10]
Training loop for a simple deep neural network to classify mnist digits 

Setup with these hyper parameters 

input dim = 784 (28 * 28 images)

hidden dim = 128 

output dim = 10 

batch size = 16 *play around with this 

learning rate = 0.001 * play around with this 

epochs = 5
